Sources/Core/Utilities/Exif/Exif.swift

```diff
@@ -77,14 +77,18 @@ public extension ExifManager {
 }
 
 // MARK: - Private Methods
-private extension ExifManager {
+private extension Exif {
   func getImageSource() -> CGImageSource? {
     var imageSource: CGImageSource?
     switch source {
     case .url(let url):
       imageSource = CGImageSourceCreateWithURL(url as CFURL, nil)
     case .data(let data):
       imageSource = CGImageSourceCreateWithData(data as CFData, nil)
+    case .image(let image):
+      if let data = image.pngData() {
+        imageSource = CGImageSourceCreateWithData(data as CFData, nil)
+      }
```

Comment:
Not sure if this will work. What happens, if the given `image` is a jpg image?
Let's remove this for now and we'll add support for "image" later.